### PR TITLE
KOTOR: Some ingame cursors

### PR DESCRIPTION
**Pull request by Nostritius**
This PR adds some ingame cursors, which will appear, when hovering over specific objects. Also i changed the click event from a mouse down to a mouse up event since this matches more the overall behaviour of the game.

### CODE REVIEW

File: src/engines/kotor/gui/gui.cpp [hunk 1]
```diff
@@ -93,11 +93,11 @@ Common::UString GUI::getName() const {
 }
 
 void GUI::mouseDown() {
-	CursorMan.setState("down");
+	CursorMan.set(CursorMan.getCurrentGroup(), "down");
```
Comment (DrMcCoy) [inline src/engines/kotor/gui/gui.cpp]: Huh...

`setState()` is supposed to do that, setting the cursor to another state in the current group.
Likewise, `setGroup()` is supposed to set a new group, with the current state.

The comments in cursorman.h even say that. However, the code does something else.
I wrote both, so I wonder what happened. :P

In either case, I think the code in cursorman.cpp should be changed to actually do what it claims to do.

**Comment by Nostritius** [comment]
The cursorman should now be fixed

File: src/engines/kotor/creature.cpp [hunk 2]
```diff
@@ -576,10 +577,12 @@ void Creature::createPC(const CharacterGenerationInfo &info) {
 }
 
 void Creature::enter() {
+	CursorMan.set("talk", CursorMan.getCurrentState());
```
Comment (DrMcCoy) [inline src/engines/kotor/creature.cpp]: That should be `CursorMan.setGroup("talk");`.

Similarly for the following two commits.

**Comment by Nostritius** [comment]
CursorMan.set should now be replaced to CursorMan.setGroup.

**Comment by DrMcCoy** [comment]
Merged, thanks! :)

**Pull request merged**